### PR TITLE
Added setting and privacy page.

### PR DESCRIPTION
**Pull request by Dea523**
Design setting and privacy page.

### CODE REVIEW

File: lib/setting_and_privacy_listtile_widget.dart [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:alpha_devayani/constants/app_heights.dart' as app_heights;
+
+class ListTileWithIconsAndContent extends StatelessWidget {
```
Comment (omkarsantoshraut) [inline lib/setting_and_privacy_listtile_widget.dart:4]: No need for this file - We can directly use the Flutter package for this type of UI.
https://pub.dev/packages/settings_ui can you try to use it directly ?

**Review by omkarsantoshraut — CHANGES_REQUESTED** [review]

**Review by abhishek2355 — APPROVED** [review]
Looking good.

File: lib/setting_and_privacy_listtile_widget.dart (hunk 1 above)
Comment (Dea523) [reply lib/setting_and_privacy_listtile_widget.dart:4]: added this pacakge. Thank you , adding this pacakage make my work easier.

Comment (omkarsantoshraut) [reply lib/setting_and_privacy_listtile_widget.dart:4]: Thanks for your changes.

Comment (omkarsantoshraut) [inline lib/setting_and_privacy_listtile_widget.dart:4]: No need of this file, right?
You could remove this one.

**Pull request merged**